avail-core/build.gradle.kts

```diff
@@ -103,6 +105,10 @@ tasks {
 		}
 	}
 
+	shadowJar {
+		dependsOn(generatePrimitivesList)
+	}
```

Comment:
This is a weaker sort of coupling than the from(...) clause on line 103.  It doesn't indicate which files form the coupling between :generatePrimitivesList and :shadowJar, so in general shadowJar wouldn't know which files to include just due to the chronological dependency.  Because of the way shadowJar does recursive class path scanning, it probably doesn't matter in this one particular case.  Oh, also shadowJar is automatically dependent on jar (which already includes All_Primitives.txt), so this is probably unnecessary anyhow.